libs/@wingcloud/framework/src/main.ts

```diff
@@ -89,13 +89,15 @@ This is a Wing app and must be run through the Wing CLI (npm install -f winglang
   const rootId = process.env.WING_ROOT_ID;
 
   const $PlatformManager = new platform.PlatformManager({ platformPaths });
+  globalThis.$PolyconFactory = $PlatformManager.createPolyconFactory();
```

Comment:
Perhaps defensively check that there isn't a factory already?

Reply:
Added